Tests/JWSRSATests.swift

```diff
@@ -63,6 +77,21 @@ class JWSRSATests: RSACryptoTestCase {
         self.performTestRSADeserialization(algorithm: .RS512, compactSerializedJWS: compactSerializedJWSRS512Const)
     }
 
+    @available(*, deprecated)
+    func testSignVerifyAndDeserializeForPS256() {
+        performTestRSASerializationValidationAndDeserialization(algorithm: .PS256)
```

Comment:
Please also add an example serialization for each of the three RSASSA-PSS algorithms, similar to what we have for the RSASSA-PKCS1-v1_5 algorithms. Then you should be able to reuse the `self.performTestRSASign` and `self.performTestRSADeserialization` helper functions. I'd prefer that instead of this new `testSignVerifyAndDeserializeForPS256 ` helper. Let me know if you disagree!

While looking at your code for `performTestRSASerializationValidationAndDeserialization` I noticed that we have an issue in `self.performTestRSASign`. See here: #189. You can just ignore that issue and use the function as you did for the RS384 test. I will fix #189 after merging your pull request.

Reply:
The thing is that the signature will differ every time since PSS isn't determinate (it stands for Probabilistic Signing Signature). This means that it cannot be tested by comparison - only by validation. My understanding is that this is part of why it is recommended over the old signing scheme.

Reply:
Oh stupid me, you're right of course.

In this case, please just remove this line from `performTestRSASerializationValidationAndDeserialization`:

``` swift
XCTAssertEqual(compactSerializedJWS, compactSerializedJWS)
```

as it doesn't really do anything (this is the same issue as in #189).